Classes/common/CDTIncrementalStore.m

```diff
@@ -16,6 +16,14 @@
 #import "CDTFieldIndexer.h"
 #import "CDTISGraphviz.h"
 
+@interface CDTISBatchUpdateResult : NSPersistentStoreResult
```

Comment:
Why isn't this `NSBatchUpdateResult`?
I think you mentioned that you needed this, but perhaps you could put a comment explaining it.